sideboard/config.py

```diff
@@ -13,6 +13,16 @@ class ConfigurationError(RuntimeError):
     pass
 
 
+def get_config_root():
+    """
+    Returns the configuration root for the system, which defaults to '/etc/sideboard'.
+    If the SIDEBOARD_CONFIG_ROOT environment variable is set, its contents will be returned instead.
+    """
+    config_root = os.environ.get('SIDEBOARD_CONFIG_ROOT', '/etc/sideboard')
+    assert os.path.isdir(config_root) and os.access(config_root, os.R_OK), '{!r} directory is not readable'.format(config_root)
```

Comment:
This seems to be failing if `/etc/sideboard` does not exist. Can we maybe do an existence check before the assert? Or, instead of failing, maybe issue a warning?